server/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const parser = require('body-parser');
+const db = require('../database/index.js');
+let port =  3030; //process.env.PORT;
```

Comment:
Are you leaving this comment in for a future refactor?

Reply:
Yes. I would like to keep it. It is something that changes with deployment that I forget.